### PR TITLE
feat: Simplify Account Key

### LINKED ISSUE #43: Simplify Account Key matching in registration emails
Currently account key is checked by looking for `CODE:{accountKeyHex}` in email headers.

Change this to use `accountkey_{accountKeyHex}`

The idea is to remove colon, so this can also be part of to address.

### PR DESCRIPTION
**Pull request by Bisht13**
Fixes #43

### CODE REVIEW

**Comment by SoraSuegami** [comment]
Thank you for your PR!
Could you fix the failing test in the contracts package?
I think it is the same error in this PR. https://github.com/zkemail/zk-regex/pull/48
Also, did your code pass the circuits test?

File: packages/relayer/src/core.rs [hunk 1]
```diff
@@ -657,7 +657,7 @@ pub(crate) async fn handle_account_transport(
     Ok(())
 }
 
-pub(crate) fn extract_account_key_from_subject(subject: &str) -> Result<String> {
+pub(crate) fn extract_account_key_from_email_address(subject: &str) -> Result<String> {
```
Comment (saleel) [inline packages/relayer/src/core.rs:660]: Function arg name should be `email_addr`.

**Review by saleel — APPROVED** [review]
LGTM 🚀 
Added a minor comment

File: packages/relayer/src/core.rs (hunk 1 above)
Comment (Bisht13) [reply packages/relayer/src/core.rs]: Fixed

**Comment by Divide-By-0** [comment]
Lots of tests are still failing...

File: packages/circuits/src/regexes/invitation_code.json [hunk 2]
```diff
@@ -2,7 +2,7 @@
     "parts": [
         {
             "is_public": false,
-            "regex_def": "CODE:0x"
+            "regex_def": "ACCOUNTKEY.0x"
```
Comment (Divide-By-0) [inline packages/circuits/src/regexes/invitation_code.json:5]: i think this should still be code?

Comment (saleel) [reply packages/circuits/src/regexes/invitation_code.json:5]: We thought of using `ACCOUNTKEY` in #43 to be more explicit - and CODE might also occur in a non-email-wallet email (extra precaution now that we don't have user<->relayer registration before a realyer can execute EmailOp).

Is `CODE` better in terms of readability? We can also have space as additional delimiter for subject? `Create Email Wallet with AccountKey 0x123` 🤔

**Review by SoraSuegami — APPROVED** [review]

**Pull request merged**